packages/adminrouter/extra/src/test-harness/tests/test_generic.config.yml

```diff
@@ -348,9 +366,21 @@ endpoint_tests:
 ######### /system/v1/logs endpoint
   - tests:
       are_upstream_req_headers_ok:
+        skip_authcookie_filtering_test: True
```

Comment:
uhhh... somehow I missed that, @mhrabovcin also pointed it out. I will grep the tests once more and try to fix all of them now.